test/system/090-events.bats

```diff
@@ -25,3 +25,35 @@ load helpers
     run_podman events --filter type=container --filter container=$cname --filter event=start --stream=false
     is "$output" "$expect" "filtering just by label"
 }
+
+@test "image events" {
+    # `remove events` seem to be broken...
+    skip_if_remote "FIXME: -remote is missing many events"
+    pushedDir=$PODMAN_TMPDIR/dir
+    mkdir -p $pushedDir
+
+    tarball=$PODMAN_TMPDIR/ball.tar
+
+    run_podman image inspect --format "{{.ID}}" $IMAGE
+    imageID="$output"
+
+    tag="registry.com/image:newtag"
+
+    run_podman push $IMAGE dir:$pushedDir
+    run_podman save $IMAGE -o $tarball
+    run_podman load -i $tarball
+    run_podman pull docker-archive:$tarball
+    run_podman tag $IMAGE $tag
+    run_podman untag $IMAGE $tag
+    run_podman tag $IMAGE $tag
+    run_podman rmi $tag
+
+    run_podman events --stream=false --filter type=image
+    is "$output" ".*image push.* $imageID dir:$pushedDir.*"
+    is "$output" ".*image save.* $imageID $tarball.*"
+    is "$output" ".*image loadfromarchive.* $tarball.*"
+    is "$output" ".*image pull.* docker-archive:$tarball.*"
+    is "$output" ".*image tag.* $imageID $tag.*"
+    is "$output" ".*image untag.* $imageID $tag.*"
+    is "$output" ".*image remove.* $imageID $tag.*"
```

Comment:
There's no order guarantee here, but I suppose that's not really worth complicating things for (as in, ensuring that the messages appear in the log in the same order as the actions).

Reply:
Good point! I'll address it. The order should be checked.